redhat-developer/app-services-cli
# fix: can now update kafka when only owner

Running `rhoas kafka update reauthentication false` was not allowed as you needed to be a org admin.

This pr allows for kafka owners to update the kafka reauthentication option and also skips prompt they are not allowed to change as they are not org admins.

### Verification Steps

1. Build main and run `rhoas kafka update reauthentication=false` when not being org admin, this will fail
2. Build this branch run `rhoas kafka update reauthentication false` and should work fine 
3. run rhoas `rhoas kafka update` and should only be able to change reauthentication an not owner
4. Do the same as org admin and you can change owner

### Type of change
